packages/shipjs/src/step/finished.js

```diff
@@ -1,3 +1,21 @@
+import chalk from 'chalk';
 import runStep from './runStep';
+import { print } from '../util';
 
-export default () => runStep({ title: 'All Finished.' }, () => {});
+export default ({ baseBranch, stagingBranch, pullRequestUrl, dryRun }) =>
+  runStep({ title: 'All Finished.' }, () => {
+    const prURL = dryRun
```

Comment:
What if we just quit when dryRun without displaying the message? 
It's because when it's dryRun, we don't even checkout the staging branch.
So this might be confusing...?

Reply:
Right, it should be hidden in dry run mode.
Should it be handled in `prepare.js`, or `finished.js`?

Reply:
I think we can just return inside finished.js.

Reply:
https://github.com/algolia/shipjs/pull/554/commits/62f810810186b32ded0a88ed4a59deed999995d3 👍